awx_collection/README.md

```diff
@@ -71,6 +71,19 @@ pip install -e .
 PYTHONPATH=awx_collection:$PYTHONPATH py.test awx_collection/test/awx/
 ```
 
+### Integration
+
+The old integration tests are kept passing, but not being expanded on.
+A master copy is hosted at: https://github.com/ansible/test-playbooks/tree/master/tower_modules
+
+A new set of integration tests as a monolithic demo is available as
+"demo" and "remove" in the `tests/integration/targets` folder. This is intended to
+server a dual-purpose as a tl;dr summary of how to use the modules.
+
+Inside the development container `dnf install ansible-test` is needed.
+Authentication setup is needed, `tower-cli login <username>`.
+Then run `make test_collection_integration`.
```

Comment:
Also need to call out `pip install junit-xml`

Reply:
@squidboylan right now this is more of a manual recipe, but we can get something like this into automation. This PR has usable stuff IMO, but I don't want to merge it without a more clear idea about where stuff will live. We have several workable options, I just want to have it all consolidated and clear.